### PR TITLE
Place units in axis labels instead of tick labels

### PR DESCRIPTION
**Pull request by briochemc**
See https://github.com/jw3126/UnitfulRecipes.jl/issues/8.

I did a few other things in this PR:
- edited the Readme with some more example uses
- added windows to the Travis CI and removed the appveyor and corresponding coverage
- changed the tests to match the changes in the code

Hopefully this is helpful 😃

### CODE REVIEW

File: src/UnitfulRecipes.jl [hunk 1]
```diff
@@ -24,34 +18,45 @@ function recipe!(attr, arrs...)
     end
 end
 
-function resolve_axis!(attr, arr::AbstractArray{<: Quantity}, axis::Int)
+"""
+    resolve_axis!(attr, arr::AbstractArray{T}, axis::Int)) where {T<:Quantity}
+
+Return `arr` data after converting it to axis unit and stripping units.
+
+Mutates `att` by converting/removing unitful attributes.
```
Comment (jw3126) [inline src/UnitfulRecipes.jl]:
```suggestion
Mutates `attr` by converting/removing unitful attributes.
```

**Review by jw3126 — COMMENTED** [review]
Thanks a lot! I like it!

**Comment by jw3126** [comment]
What does this PR do to an axis without units? e.g. `plot(randn(10)*m, randn(10))` ?

**Comment by briochemc** [comment]
It does what you expect I guess? This is the output for me:

example5

**Comment by jw3126** [comment]
Ah right, I was confused!

**Comment by jw3126** [comment]
Can you bump to version `v0.1.2`, then I will tag a release after this PR.

**Comment by briochemc** [comment]
Oh yes I forgot! Here it is 😄

**Review by jw3126 — APPROVED** [review]

**Review by jw3126 — COMMENTED** [review]
@JuliaRegistrator register

File: Project.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name = "UnitfulRecipes"
 uuid = "42071c24-d89e-48dd-8a24-8a12d9b8861f"
 authors = ["Jan Weidner"]
-version = "0.1.1"
```
Comment (jw3126) [inline Project.toml:4]: @JuliaRegistrator register

**Review by jw3126 — COMMENTED** [review]
@JuliaRegistrator register

**Comment by jw3126** [comment]
dang I always forget in what way to comment a commit in order for JuliaRegistrator to listen

**Comment by briochemc** [comment]
Haha same I always have to go and check the package and copy-paste it 😄

**Comment by briochemc** [comment]
Anyway thanks to your reactivity contributing to this was very satisfying! So yeah thanks!

**Comment by jw3126** [comment]
Thanks a lot for the contribution! I appreciate it!

**Pull request merged**